modules/nw-configing-high-performance-multicast-with-sriov.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * networking/configureing-high-performance-multicast-with-SRIOV.adoc
+
+= About high performance multicast
+
+Openshift SDN support the multicast between Pods on the default network, however
+that is best used for low-bandwidth coordination or service discovery, and not a 
+high-bandwidth solution. For applications of streaming media, such as IPTV and
+multipoint videoconferencing, you can utilize SR-IOV to provide near-native 
+performance.
+
+When using SRIOV additional interface for multicast:
+
+* Multicast packages sent or received by a Pod through the additional SR-IOV
+interface.
+* The physical network which connects the SR-IOV interfaces decides the
+multicast routing and topology, which is not controlled by Openshift.
+
+= Using SR-IOV interface for multicast
+
+Take following procedure as an example.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster with a user that has the `cluster-admin` role.
+
+.Procedure
+. Create a SriovNetworkNodePolicy CR
+
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-example
+  namespace: openshift-sriov-network-operator
```

Comment:
Is that namespace ('openshift-sriov-network-operator') fixed value?
If so, we may need to describe as note or some small section otherwise user may use original namespace for that.

Reply:
Thanks for the comments. The namespace is fixed. It is mentioned in the general SRIOV configuration document.

Reply:
Thank you for the reply! Got it.